SPECS/cni-plugins/cni-plugins.spec

```diff
@@ -1,8 +1,8 @@
 %define _default_cni_plugins_dir /opt/cni/bin
 Summary:        Container Network Interface (CNI) plugins
 Name:           cni-plugins
-Version:        0.9.1
-Release:        14%{?dist}
+Version:        1.3.0
```

Comment:
We have missed to update the relevant entry in `cgmanifest.json`

Reply:
Ops, thanks for pointing out. I had it done locally, but I forgot to push it :)